Xamarin.Forms.Core.UnitTests/BindingUnitTests.cs

```diff
@@ -1897,6 +1897,11 @@ public void ConvertIsCultureInvariant (string culture)
 			slider.Value = 0.9;
 
 			Assert.That (vm.Text, Is.EqualTo ("0.9"));
+
+			// people write numbers using the separator accepted in their locale
+			var ds = System.Threading.Thread.CurrentThread.CurrentCulture.NumberFormat.NumberDecimalSeparator;
+			vm.Text = $"0{ds}7";
+			Assert.That(slider.Value, Is.EqualTo(0.7));
```

Comment:
This test explicitly expects the invariant culture is used when displaying the slider value. This would lead me to believe the original developer thought about Culture and decided to go with the invariant culture everywhere. So this PR is less of a "fix" and more of a feature request; The current behavior is "expected" from the perspective of the original developer. As a feature request, the work item would be to review _all_ renderers and re-establish a new expected behavior wrt culture -- which is to say that it should respect whatever culture the main thread is using.